cmd/crawler/main.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+    "fmt"
+    "strings"
+    "bufio"
+    "os"
+    "time"
+
+    "github.com/gocolly/colly/v2"
+)
+
+func main() {
+    fmt.Println("Go go yees")
+    c := colly.NewCollector()
+
+    used := make(map[string]bool)
+    urlQueue := make([]string, 0)
+    articles := make([]string, 0)
+
+    c.OnHTML("a[href]", func(e *colly.HTMLElement) {
+        url := e.Attr("href")
+        if !used[url] && strings.Contains(url, "arxiv.org") {
+            if strings.Contains(url, "abs") {
```

Comment:
You'd better move all your constant strings to a config with some meaningful descriptions (hostname, topic etc).

Reply:
Done